package.json

```diff
@@ -60,6 +67,7 @@
     "lint-staged": "^15.2.2",
     "npm-run-all": "^4.1.5",
     "prettier": "^3.2.5",
+    "tailwindcss": "^3.4.14",
```

Comment:
The reason is that I personally implement a majority of the styles across all of our UIs. I'm sure if you ran a git blame across all of our CSS I would be personally responsible for 50-75%  

Front-end frameworks generally: 
1. Go out of style quickly
2. Get in the way of implementation, especially for me 
3. Decrease render performance noticeably

Reply:
> The reason is that I personally implement a majority of the styles across all of our UIs.

If we had a design system (with UI components, fonts, colors, etc...) it'd allow all contributors to proper style UI related projects without a constant css copy/pasting from other projects or introducing new CSS frameworks.

Regarding this PR, it's too late to remove `tailwindcss` and `daisyui` because it requires too much code refactor.